index.html

```diff
@@ -1,8 +1,45 @@
-<!DOCTYPE html>
-<head>
-
-</head>
-<body>
-    <p>GH Action Test #2</p>
-</body>
-</html>
\ No newline at end of file
+<script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
+
+<div id="app">
+    <h1>HP & WSU-V Capstone</h1>
+    <form @submit.prevent="createJob">
+        <label for="jobTitle">Job Title </label>
+        <input type="text" id="jobTitle" name="jobTitle" v-model="jobTitle" />
+        <label for="pageCount">  Page Count </label>
+        <input type="text" id="pageCount" name="pageCount" v-model="pageCount" />
+        <button type="submit">Submit</button>
+    </form>
+    <p>{{message}}</p>
+    
+</div>
+
+<script>
```

Comment:
In the future, scripts won't live in `index.html`, but in a separate `*.js` file that we import in. For now, this works fine.